src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -92,10 +84,7 @@
                source_package: source_package.name)
       end
 
-      before do
-        bs_request.staging_project = staging_project
```

Comment:
I check again that requests are «ready» whether they are associated to a staging project or not, as it was before, but using `bs_request_2` instead of `bs_request` which is always rejected because it is not in state `new`.

We also realized that the test was wrong. Now it is ok, «ready» requests are those which state `new` and all its reviews `accepted`. The solution is a bit tricky so please have another look and give some suggestions if needed. @Ana06 @bgeuken @coolo

Reply:
Right now the test fails for me with `Review::NotFoundError`

Reply:
@bgeuken rebasing [this PR](https://github.com/openSUSE/open-build-service/pull/6721) is causing the failures... working on this...